includes/post-functions.php

```diff
@@ -139,20 +139,30 @@ function today_get_post_meta_info( $post ) {
  * @return string Author bio markup
  */
 function today_get_post_author_bio( $post ) {
-	$author_bio = trim( get_field( 'post_author_bio', $post ) );
+	$author_data = today_get_post_author_data( $post );
+	$author_bio  = $author_data['bio'] ?? null;
 
 	ob_start();
 	if ( $author_bio ) :
-		$author_byline     = get_the_author();
-		$author_title      = get_field( 'post_author_title', $post );
-		$author_photo_data = get_field( 'post_author_photo', $post );
+		$author_byline     = $author_data['name'];
+		$author_title      = $author_data['title'];
+		$author_photo_data = $author_data['photo'];
```

Comment:
It's theoretically possible for the array to be returned without these indexes, so we either need to preform an index check here, or else in the `today_get_post_author_data` function, provide an empty implementation (all indexes are defined, but have a null value).

Reply:
Null checks added 👍